File: ruby/fixtures/basics/echo_fixture.rb
module Fixtures
  module Basics
    class EchoFixture
      def echo_string(string)
        string
      end
      
      def echo_int(value)
        return value.to_i;
      end
      
      def echo_float(value)
        return value.to_f;
      end
      
      def echo_double(value)
        return value.to_f;
      end
    end
  end
end

File: javascript/fixtures/basics/EchoFixture.js
fixtures.basics.EchoFixture = function() {
    this.echoString = function(string) {
        return string;
    }
    
    this.echoInt = function(value) {
        return parseInt(value);
    }
    
    this.echoDouble = function(value) {
        return JsSlim.Converter.floatToString(parseFloat(value));
    }
    
    this.echoFloat = function(value) {
        return this.echoDouble(value);
    }
}
